app/workers/education_form/create_daily_spool_files.rb

```diff
@@ -90,8 +92,10 @@ def format_application(form)
       # the spool file has a requirement that lines be 80 bytes (not characters), and since they
       # use windows-style newlines, that leaves us with a width of 78
       wrapped = word_wrap(@application_template.result(binding), line_width: 78)
+      # We can only send ASCII, so make a best-effort at that.
+      transliterated = Iconv.iconv('ascii//translit', 'utf-8', wrapped).first
```

Comment:
Except that `force_encoding` does not handle the use case in the test, which is keeping the intent of the character entered by the user as best as possible, rather than discarding it.